testing/jormungandr-scenario-tests/src/scenario/controller.rs

```diff
@@ -166,6 +186,7 @@ impl Controller {
         working_directory: ChildPath,
         blockchain: Blockchain,
         topology: Topology,
+        node_key_files: HashMap<String, NamedTempFile>,
```

Comment:
Same issue as with the other controller. Maybe there should be better code reuse as well.